aztk/client/base/helpers/get_application_log.py

```diff
@@ -55,23 +66,37 @@ def get_log_from_storage(blob_client, container_name, application_name, task):
     return models.ApplicationLog(
         name=application_name,
         cluster_id=container_name,
-        application_state=task.state.name,
+        application_state=task.state,
         log=blob.content,
         total_bytes=blob.properties.content_length,
-        exit_code=task.execution_info.exit_code,
+        exit_code=task.exit_code,
     )
 
 
-def get_log(batch_client, blob_client, cluster_id: str, application_name: str, tail=False, current_bytes: int = 0):
+def wait_for_scheduling_target_task(base_operations, cluster_id, application_name):
+    application_state = base_operations.get_task_status(cluster_id, application_name)
+    while TaskState(application_state) not in [TaskState.Completed, TaskState.Failed]:
+        time.sleep(3)
+        # TODO: enable logger
```

Comment:
todo?

Reply:
I'm going to hold off on logging in the sdk for this PR. That should probably come in another PR. I just wanted to leave a couple of these as reminders.